common/check_report_used_space

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# Copyright European Organization for Nuclear Research (CERN) 2013
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Donata Mielaikaite, <donata.mielaikaite@cern.ch>, 2020
+
+'''
+Probe to check used space.
+'''
+
+import datetime
+import sys
+import traceback
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if __name__ == "__main__":
+    try:
+        for rse in list_rses():
+            limits = get_rse_usage(rse['id'])
+            for usage in limits:
+                if usage['source'] == 'rucio':
+                    monitor.record_gauge(stat='judge.used_space.%s' %rse['rse'], value=usage['used'])
```

Comment:
We should record a gauge per usage source per RSE

Reply:
I don't get what you mean by 'gauge per usage source per RSE'. How's that different from what there is already in this probe? 
https://github.com/rucio/rucio/blob/master/lib/rucio/core/rse.py#L680

Reply:
There are some usages with `usage['source']` different from `rucio`, we should report those as well, e.g. `expired` and `obsolete`. I would make new stats for each.

Reply:
Stats were added.